### ISSUE TITLE
Question: How do I specifically turn on browserField for particular requires?

### ISSUE THREAD

**Issue by shamasis** [substack/node-browserify#1676]
I am programatically generating a bundle:

```
var browserify = require('browserify'),
    bundle = browserify({
        insertGlobalVars: false,
        detectGlobals: true,
        browserField: false,
        bare: true,
        builtins: false,
        commondir: true
    });

bundle.require('crypto-browserify', {
    expose: 'crypto',
    browserField: true // THIS IS WHAT I WANT TO DO
});
```

In the above code, I want to bundle all code with browserField turned off, but only turn it on for the particular `crypto-browserify` require. How do I do it? I may have missed something in documentation, but please excuse me for that.

**Comment by shamasis** [comment]
@substack sorry for pinging directly. But this is kind of urgent. Any ideas? Anything using transformations or plugins?

**Comment by shamasis** [comment]
Anybody? I am thinking of writing a transformer that will for a set of requires, replace require with whatever is returned via browser-resolve module.

**Comment by dotansimha** [comment]
Same issue here. @shamasis, any idea?

**Comment by shamasis** [comment]
I had given myself a full week to see if anyone from community can show me a direction before I deep dive. 

My plan otherwise is to write a transformer that takes in package names and when those requires are encountered, return the browser-require result instead of original.

**Comment by snajjar** [comment]
@shamasis Have you written your transformer? I'll need it to, actually.

**Comment by shamasis** [comment]
Yep. I’ve accommodated basics in github.com/postmanlabs/postman-sandbox but didn’t get time to extract it out as an independent module
